packages/feedback-react/src/questions/SliderQuestion.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ChangeEventHandler, useEffect, useState } from "react";
+import { Slider } from "@fremtind/jkl-slider-react";
+
+import { useFollowUpContext } from "../followup/followupContext";
+import { useMainQuestionContext } from "../main-question/mainQuestionContext";
+import { QuestionProps } from "../types";
+
+export const SliderQuestion: React.VFC<QuestionProps> = ({ label, name, options }) => {
+    const numbers = options?.map((option) => parseInt(option.value.toString()));
+    const from = numbers?.reduce((number, lowest) => (number < lowest ? number : lowest));
```

Comment:
Tror det kan være lurt å navngi funksjonene i reducerne for å gjøre det mer lesbart.
numbers.reduce(findMinValue(a,b)) eller noe sånt

Reply:
Denne er jeg mer med på! Reducere er alltid stress å følge med på 😅